superchain/configs/sepolia-dev-0/superchain.yaml

```diff
@@ -7,6 +7,7 @@ l1:
 protocol_versions_addr: "0x252CbE9517F731C618961D890D534183822dcC8d"
 superchain_config_addr: "0x02d91Cf852423640d93920BE0CAdceC0E7A00FA7"
 
-canyon_time: 1698436800  # Fri Oct 27 20:00:00 UTC 2023
-delta_time: 1706555000   # Mon Jan 29 19:03:20 UTC 2024
-ecotone_time: 1706634000 # Tue Jan 30 17:00:00 UTC 2024
+canyon_time: 1698436800  # Fri 27 Oct 2023 20:00:00 UTC
+delta_time: 1706555000   # Mon 29 Jan 2024 19:03:20 UTC
```

Comment:
Let's actually set Canyon and Delta to 0 in this superchain config, and overwrite both values to these times in `base-devnet-0`. The reason is that any future chains joining this superchain will have Canyon and Delta activated at genesis already.

Reply:
[c2e25cc](https://github.com/ethereum-optimism/superchain-registry/pull/101/commits/c2e25ccf4d85f3a39d028452976c61696d82e093)